docusaurus/docs-internals/control-arguments/css.md

```diff
@@ -2,4 +2,152 @@
+The Brizy's `CSS` control empowers users with the ability to dynamically customize the appearance of elements on their website without the need for coding expertise. With this feature, users can effortlessly adjust various CSS properties, such as colors, fonts, margins, and more, to achieve their desired visual style .
+
+It supports two approaches: using `selector` for predefined CSS properties in combination with others controls and `style` for custom CSS generation.
+
+### Selector
+The `selector` key work in conjunction with other controls to apply predefined CSS properties to the selected element. The Brizy's controls that allow this functionality are : 
+- `backgroundColor`
+- `border`
+- `boxShadow`
+- `colorPicker`
+- `corners`
+- `filters`
+- `margin`
+- `padding`
+- `textShadow`
+- `typography`
+
+#### Usage
+
+```js
+ {
+  id: "backgroundCSS",
+  type: "backgroundColor",
+  states: [NORMAL, HOVER],
+  selector: "{{WRAPPER}}:hover .brz-video-content"
+}
+```
+
+This code will apply the background color to the selected element with the selector `.brz-video-content` when the user interacts with the control. It will generate a similar CSS output as below:
+```css
+.brz .brz-css-gpWLU .brz-video-content {
+  background-color: #000000;
+}
+
+.brz .brz-css-gpWLU:hover .brz-video-content {
+  background-color: #ffffff;
+}
+```
+
+There are three states that can be used with the `selector` key: `NORMAL`, `HOVER`, and `ACTIVE`. The `NORMAL` state is the default state of the control, while the `HOVER` and `ACTIVE` states are used when the user interacts with the control.
+If the `states` array contains `ACTIVE` or `HOVER` but in the selector they won't be provided, then the control will add the pseudo class `:hover` (for `HOVER`) at the end of selector and  the suffix `".active"`(for `ACTIVE`) to the selector.
+
+```js
+ {
+  id: "backgroundCSS",
+  type: "backgroundColor",
+  states: [NORMAL, ACTIVE, HOVER],
+  selector: "{{WRAPPER}}:hover .brz-video-content"
+}
+```
+
+The CSS output will be like below:
+```css
+.brz .brz-css-gpWLU .brz-video-content {
+  background-color: #000000;
+}
+
+.brz .brz-css-gpWLU .brz-video-content.active {
+  background-color: #ffffff;
+}
+
+.brz .brz-css-gpWLU .brz-video-content:hover {
+  background-color: #ff0e0e;
+}
+```
+
```

Comment:
1. need to add also an example when we use custom ":hover" and ".active" in selector, as example:

`{{WRAPPER}} .brz-list:hover .brz-list-item.active`

2. add also examples with more selector:
```ts
{
  id: "asd"
  type :"ddd",
  selector: ".brz-test1, .brz-test2, #brz-iframe123"
}
``